data/dashboard.yml

```diff
@@ -1,7 +1,6 @@
 # These sections are magically populated from repos.yml
```

Comment:
I'm going to remove this misleading line! The sections do seem to be curated manually 😅

Reply:
https://github.com/alphagov/govuk-developer-docs/pull/3407